include/circt/Firtool/Firtool.h

```diff
@@ -363,6 +364,7 @@ class FirtoolOptions {
   std::string chiselInterfaceOutDirectory;
```

Comment:
Just pattern-matching, looks like some options have `set` methods as well -- should this be exposed there?
Since I think we're hoping to drop and delete the pass soon anyway not sure, but just an observation :+1: .

Reply:
(Maybe only if want to be part of the CAPI)

Reply:
No, this pass was a sifive-internal one, so we shouldn't expose it to other users of the C API.

Reply:
:100: good call, thanks!